apps/github-cascading-app/eslint.config.mjs

```diff
@@ -1,7 +1,7 @@
-import local from './eslint.local.config.mjs';
 import shared from '../../eslint.shared.config.mjs';
+import local from './eslint.local.config.mjs';
 
 export default [
   ...shared,
-  ...local
+  ...local,
```

Comment:
I don't think the trailing comma on an array is a good idea.
I understand it can useful on `import` but I am not sure we want to enforce it on const object or array